### DISCUSSION TITLE
WLAN Sendeleistung einstellen

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2432] [Q&A]
Hallo,

bei meiner openDTU Fusion und auch der onBattery kann man ja unter "DTU Konfiguration" die Sedeleistung einstellen, bei mir für CMT2300A. Diese Einstellung wirkt sich aber nur auf die Verbindung openDTU => WR aus.

Ich habe hier die Situation, dass die openDTU unmittelbar neben der FritzBox hängt und somit die 2,4 GHz WLAN-Verbindung mit Vollgas in die FB bläst. Das ist dann ein Wert, der mindestens 45 dBm über allen anderen Werten dort liegt. Da weiß ich nicht, ob das immer ohne Probleme abläuft, vielleicht wird ja auch direkt in die FB eingestrahlt, sind ja nur ca. 70 cm Abstand. Versetzen der DTU kann ich schlecht wegen der Platzverhältnisse.

Kennt jemand von Euch  eine Möglichleit, die 2,4 GHz Sendeleistung für das WLAN in der openeDTU zu reduzieren?
Leider habe ich da bisher noch nichts gefunden.

Vielen Dank im Voraus und Grüße
Werner

**Comment by stefan123t** [comment]
This may be possible using esp-idf function:
esp_wifi_set_max_tx_power(int8_t power);

https://github.com/espressif/esp-idf/blob/1c7a8b3b712c4020562551f692d0ced8f7470d2c/components/esp32/include/esp_wifi.h#L892-L919

This API sets maximum WiFi transmiting power. Values got from power are mapped to transmit power levels as follows:
  * - 78: 19.5dBm
  * - 76: 19dBm
  * - 74: 18.5dBm
  * - 68: 17dBm
  * - 60: 15dBm
  * - 52: 13dBm
  * - 44: 11dBm
  * - 34: 8.5dBm
  * - 28: 7dBm
  * - 20: 5dBm
  * - 8:  2dBm
  * - -4: -1dBm

**Comment by Fokker-1** [comment]
Hello

Thanks for your information.
But I don't have sufficient knowledge in programming to do this, unfortunately.

Reducing WiFi transmiting power helps to reduce power consumption, too.

Best regards
Werner
